management-gateway/src/main/java/org/frankframework/management/security/JwtSecurityFilter.java

```diff
@@ -75,7 +75,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 			securityContextHolderStrategy.setContext(newContext);
 		} catch (IOException e) {
 			this.securityContextHolderStrategy.clearContext();
-			log.debug("Failed to process authentication request", e);
+			log.warn("failed to process authentication request", e);
 			throw e;
```

Comment:
The exception is also thrown again, so it is not logged or processed as a failure/warning at higher levels?

Reply:
ServletExceptions are 'unmanaged' in the sense that the container either throws or suppresses them. This way they (at least) appear in our logs.